lms/envs/aws.py

```diff
@@ -605,3 +605,5 @@
 PROFILE_IMAGE_SECRET_KEY = AUTH_TOKENS.get('PROFILE_IMAGE_SECRET_KEY', PROFILE_IMAGE_SECRET_KEY)
 PROFILE_IMAGE_MAX_BYTES = ENV_TOKENS.get('PROFILE_IMAGE_MAX_BYTES', PROFILE_IMAGE_MAX_BYTES)
 PROFILE_IMAGE_MIN_BYTES = ENV_TOKENS.get('PROFILE_IMAGE_MIN_BYTES', PROFILE_IMAGE_MIN_BYTES)
+if FEATURES['IS_EDX_DOMAIN']:
```

Comment:
@chrisndodge I believe this is the fix that I put in that caused microsites to show the Xsy logo. What is the better way to test that this is a true edX domain site but not a microsite? I was under the impression that we had a lot of code that looks at this variable to add in edX branding, so there may be other places that might need fixing too.